context/app/static/js/components/Detail/ProvTabs/ProvTabs.jsx

```diff
@@ -27,30 +36,34 @@ function ProvTabs(props) {
         aria-label="Detail View Tabs"
         TabIndicatorProps={{ style: { backgroundColor: '#9CB965' } }}
       >
-        <StyledTab label="Table" id="tab-0" aria-controls="tabpanel-0" />
-        <StyledTab label="Graph" id="tab-1" aria-controls="tabpanel-1" />
-        {shouldDisplayDag && <StyledTab label="Analysis Details" id="tab-2" aria-controls="tabpanel-2" />}
+        {shouldDisplayTable && <StyledTab label="Table" id="tab-table" aria-controls="tabpanel-0" />}
+        <StyledTab label="Graph" id="tab-graph" aria-controls={`tabpanel-${graphIndex}`} />
```

Comment:
Still doing the integer indexes in `aria-controls`: Is that necessary?

Reply:
I honestly don't know. I think this was pulled from a material ui example and thought it best to keep since we're in a time crunch.